openlibrary/plugins/openlibrary/js/index.js

```diff
@@ -88,6 +90,7 @@ jQuery(function () {
     addFadeInFunctionsTojQuery($);
     jQueryRepeat($);
     initAnalytics($);
+    WikipediaCitationButtonPressed($);
```

Comment:
Just write  `$('#wikiselect').on('focus', function(){$(this).select();})`.
Let's avoid the import (see below)

Reply:
Oh, I thought of importing only because I felt `index.js` only took imports and had no code of its own. Fixing this and commit-ing in a minute after testing.